articles/users/index.md

```diff
@@ -0,0 +1,19 @@
+---
+classes: topic-page
+title: Users
+description: Learn about working with users in Auth0
+---
+<!-- markdownlint-disable MD041 MD002 -->
+<div class="topic-page-header">
+  <div data-name="example" class="topic-page-badge"></div>
+  <h1>Users</h1>
+  <p>
+    Learn about working with users in Auth0.
+  </p>
+</div>
+
+<%= include('../_includes/_topic-links', { links: [
```

Comment:
You can add also all of the following I think:

![image](https://user-images.githubusercontent.com/11715799/32221512-785e9aa8-be3e-11e7-9752-deac2a2111f1.png)

I'm also thinking that this index could be the landing page when you click Articles > User Management. Wdyt?

Reply:
Add also this doc, it's a new one that Jerrie wrote and got deployed yesterday https://auth0.com/docs/connections/database/migrating-okta

Reply:
Looks good but the list is long. How about grouping them?

For example:
- User Profile
  - User Profile
  - Structure of the User Profile
  - User Picture
  - Metadata
  - etc
- Import/Export
  - Bulk Import Users to Auth0
  - Migrate Users from Okta to Auth0
- etc